scripts/setup/create_singlenode_container.sh

```diff
@@ -37,14 +37,19 @@ server_exec() {
     # Spin up vHive under container mode.
     server_exec 'sudo DEBIAN_FRONTEND=noninteractive apt-get autoremove' 
     server_exec "git clone --branch=$VHIVE_BRANCH https://github.com/ease-lab/vhive"
-    server_exec 'cd vhive; ./scripts/cloudlab/setup_node.sh stock-only'
+    
+    server_exec 'sudo wget -q https://github.com/mikefarah/yq/releases/download/v4.27.3/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq'
+    server_exec '~/invitro/scripts/setup/rewrite_yaml_files.sh'
```

Comment:
We should run this file after we cloned the loader. And the name of the folder is loader, not the invitro.